apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/Form.tsx

```diff
@@ -0,0 +1,150 @@
+        options={[
+          {
+            id: 'not-contacted',
+            label: 'Not contacted',
+            value: 'notContacted',
+          },
+          {
+            id: 'contacted',
+            label: 'Contacted',
+            value: 'contacted',
+          },
+          {
+            id: 'in-process',
+            label: 'In process',
+            value: 'inProcess',
+          },
+          {
+            id: 'no-match',
+            label: 'No match',
+            value: 'noMatch',
+          },
+          {
+            id: 'not-interested',
+            label: 'Not interested',
+            value: 'notInterested',
+          },
+          {
+            id: 'hired',
+            label: 'Hired',
+            value: 'hired',
+          },
+        ]}
+      />
+      <div className="flex space-x-4">
+        <TextEntry
+          initialValue={firstName}
+          placeholder="Guy"
+          inputId="linkedin-first-name"
+          handleChange={(e) => setFirstName(e.target.value)}
+          labelText="First Name"
+        />
+        <TextEntry
+          initialValue={lastName}
+          placeholder="Tarenbois"
+          inputId="linkedin-last-name"
+          handleChange={(e) => setLastName(e.target.value)}
+          labelText="Last Name"
+        />
+      </div>
+      <TextEntry
+        initialValue={jobTitle}
+        placeholder="Musicien"
+        inputId="linkedin-job-title"
+        handleChange={(e) => setJobTitle(e.target.value)}
+        labelText="Job title"
+      />
+      <TextEntry
+        initialValue={currentCompany}
+        placeholder="Rock Band"
+        inputId="linkedin-current-company"
+        handleChange={(e) => setCurrentCompany(e.target.value)}
+        labelText="Current company"
+      />
+      <TextEntry
+        initialValue={location}
+        placeholder="Paris"
+        inputId="linkedin-location"
+        handleChange={(e) => setLocation(e.target.value)}
+        labelText="Location"
+      />
+      <SelectEntry
+        labelText="Gender"
+        id="gender"
+        handleChange={(e) => setGender(e.target.value)}
+        initialValue={gender}
+        options={[
+          {
+            id: 'empty-gender',
+            label: '',
+            value: '',
+          },
+          {
+            id: 'female',
+            label: 'Female',
+            value: 'female',
+          },
+          {
+            id: 'male',
+            label: 'Male',
+            value: 'male',
+          },
+          {
+            id: 'other',
+            label: 'Other',
+            value: 'other',
+          },
+        ]}
+      />
+      <TextAreaEntry
+        inputId="comment"
+        labelText="Comment"
+        value={comment}
+        handleChange={(e) => setComment(e.target.value)}
+      />
+      <div className="flex space-x-2">
+        <ButtonPrimary className="flex-grow">Save</ButtonPrimary>
+        <ButtonPrimary onClick={onReload}>🔄</ButtonPrimary>
```

Comment:
Did you add this button from the start or did you encounter an edge case that spurred its creation?

Reply:
First because I happened to have issues with the scraper being a bit slow on some values were empty. This button was a way to relaunch the scraper once the page was fully loaded. But once this issue fixed directly in the scraper, I kept the button as I found it super handy to go back to linkedin iso when playing with the form. I really see an added value for the user, what do you think ?

Reply:
I agree, nice one. Let's keep it!